frontend/src/libs/components/search/libs/constants.ts

```diff
@@ -0,0 +1,5 @@
+const DEFAULT_SEARCH_PAYLOAD: { search: string } = {
```

Comment:
Create a separate type for this, I've seen you use it in multiple places

Reply:
done